connecting-to-testnet/building-binary.md

```diff
@@ -1,66 +1,67 @@
 ---
-order: 1
+order: 2
 ---
 
 # Building and Starting a Full Node
 
-## Requirements
-
-* [Go](https://golang.org/) (version 1.15 or higher)
-* [make](https://en.wikipedia.org/wiki/Make_(software))
-* [jq](https://github.com/stedolan/jq)
-* [git](https://git-scm.com/)
-* [curl](https://github.com/curl/curl)
-
+To build and start a full node, you build the binary, and then initialize and start a node. 
 
 ## Building the Binary
 
-Clone the code repository:
+This testnet repo is `b-harvest/gravity-dex`. This repo provides the functionality of: 
 
-```
-git clone --branch v1.0.1 https://github.com/b-harvest/gravity-dex
-```
+- `cosmos/gaia` the source code behind Cosmos Hub
+- `tendermint/liquidity` the module source code of Gravity DEX 
 
-We're using `b-harvest/gravity-dex`, which is `cosmos/gaia` (the source code behind Cosmos Hub) with the `tendermint/liquidity` module (source code of Gravity DEX) imported.
+1. Clone the code repository:
 
-Install node's binary `gaiad`:
+    ```
+    git clone --branch v1.0.1 https://github.com/b-harvest/gravity-dex
+    ```
 
-```
-cd gravity-dex && make install
-```
+1. Install the node's binary `gaiad`:
+
+    ```
+    cd gravity-dex && make install
+    ```
 
 ## Initializing a Node
 
-Initialize your node (replace `YOUR_MONIKER` with any string that identifies your node):
+To initialize your node:
 
 ```
 gaiad init YOUR_MONIKER --chain-id cosmoshub-testnet
 ```
+ 
+where `YOUR_MONIKER` is a placeholder for a string that identifies your node. Be sure to replace `YOUR_MONIKER` with a self-describing string for your node. 
 
 ## Starting a Node
 
-Download the genesis file and save it into the data directory of your node:
+1. Download the genesis file:
 
-```
-curl -s https://rpc.testnet.cosmos.network/genesis | jq -r .result.genesis > ~/.gaia/config/genesis.json
-```
+    ```
+    curl -s https://rpc.testnet.cosmos.network/genesis | jq -r .result.genesis > ~/.gaia/config/genesis.json
+    ```
+1. Save the genesis file in the data directory of your node.
 
-Add seed nodes:
+1. Add seed nodes using the command that is suitable for your environment. 
 
-```
-sed -i 's/seeds =.*/seeds = "30dce239c56cfd25cf0cebf5726c7c8be1e46f66@104.196.38.186:31819,754a5f864adc5a60b287d4aed4f0ab11d8b056c8@34.73.24.113:31654"/g' ~/.gaia/config/config.toml
-```
+    - For operating systems other than macOS or BSD:
 
-If you're using macOS or BSD, use the following command:
+    ```
+    sed -i 's/seeds =.*/seeds = "30dce239c56cfd25cf0cebf5726c7c8be1e46f66@104.196.38.186:31819,754a5f864adc5a60b287d4aed4f0ab11d8b056c8@34.73.24.113:31654"/g' ~/.gaia/config/config.toml
+    ```
 
-```
-sed -i '' 's/seeds =.*/seeds = "30dce239c56cfd25cf0cebf5726c7c8be1e46f66@104.196.38.186:31819,754a5f864adc5a60b287d4aed4f0ab11d8b056c8@34.73.24.113:31654"/g' ~/.gaia/config/config.toml
-```
+    - For macOS or BSD operating systems:
```

Comment:
do we need to list BSD since its deprecated?

Reply:
Oh, but it hasn't! From Wikipedia:

> The term "BSD" commonly refers to its descendants, including FreeBSD, OpenBSD, NetBSD, and DragonFly BSD.

We don't have to mention it, though, not sure if it's even supported.